### PR TITLE
Add catch for too many private recipients

### PR DESCRIPTION
**Pull request by stripedpajamas**
Addresses #838 

Looks like this (changes welcome):

too_many_recps

### CODE REVIEW

File: modules/message/sheet/preview.js [hunk 1]
```diff
@@ -21,6 +21,24 @@ exports.create = function (api) {
       var isPrivate = msg.value.private
       var isRoot = !msg.value.content.root
       var recps = (msg.value.content.recps || []).filter(id => id !== msg.value.author)
+
+      // handle too many private recipients
+      if (isPrivate && recps.length > 7) {
+        return {
+          content: [
+            h('h2', ['Too many recipients']),
```
Comment (mmckegg) [inline modules/message/sheet/preview.js]: These strings should use `i18n` to make them localisable. It should just be a matter of wrapping with a call to `i18n`, for example:

```
h('h2', [i18n('Too many recipients')])
```

But you'll need to make sure i18n is included in scope (I think it is in this file).

File: modules/message/sheet/preview.js [hunk 2]
```diff
@@ -21,6 +21,24 @@ exports.create = function (api) {
...
+            h('div.info', [
+              h('p', [
+                'Private messages can only be addressed to up to 7 people. ',
+                'Your message has ' + recps.length + ' recipients. '
+              ]),
+              h('p', ['Please go back and remove some of the recipients.'])
+            ])
+          ],
+          classList: ['-private'],
+          footer: [h('button -cancel', { 'ev-click': cancel }, i18n('Cancel'))]
```
Comment (mmckegg) [inline modules/message/sheet/preview.js]: I think this button "Cancel", should be called "Close".

**Comment by Serkan-devel** [comment]
but why should recipes be limited?

**Comment by stripedpajamas** [comment]
@Serkan-devel see #838 (specifically the response from cryptix) for more info. It is just part of the protocol.

**Pull request merged**